scheduler-jobs/README.md

```diff
@@ -0,0 +1,17 @@
+# How-To Run Locally:
+
+To run locally as a process: `go run scheduler-jobs.go`, assuming you have a scheduler and dapr running accordingly (see below).
+
+Run Scheduler:
+![scheduler_debugger_config.png](scheduler_debugger_config.png)
+
+Run Dapr sidecar:
+![sidecar_debugger_config.png](sidecar_debugger_config.png)
+
+To run locally as a container:
```

Comment:
Wouldn't we need additional networking config (`host.docker.internal`?) for an app running in a docker container to reach the daprd and scheduler processes running locally?